generators/client/templates/react/src/main/webapp/app/shared/layout/header/header.tsx

```diff
@@ -11,7 +11,8 @@ import {
   FaList, FaTasks, FaDashboard, FaBook, FaWrench, FaSignIn, FaSignOut,
   FaClockO, FaHddO,
   // tslint:disable-next-line
-  FaRoad, FaAsterisk, FaBell
+  FaRoad,
+  /* FaAsterisk, */ FaBell
```

Comment:
why is FaAsterisk commented out, also dont break line as we want `// tslint:disable-next-line` to work here

Reply:
We had to add " // tslint:disable-next-line " above each faRoad, FaAsterisk and FaBell in next commit in order to pass the lint tests